hyprland-share-picker/mainpicker.ui

```diff
@@ -6,10 +6,16 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>500</width>
-    <height>290</height>
+    <width>800</width>
```

Comment:
can we leave the default size be? I find the new one pretty huge.

Reply:
I will make the dialog smaller by default but now is gonna be 500x300 because the layout is a bit different i works better with that extra size

Also, do you mind if i also implement the restoring of the dialog size?
I hate to have to resize the dialog each time